openapi.yaml

```diff
@@ -164,353 +102,495 @@ paths:
           schema:
-            $ref: '#/definitions/NewSSNResultObject'
-        - $ref: '#/parameters/osuID'
-        - $ref: '#/parameters/pretty'
-        - $ref: '#/parameters/authorization'
+            $ref: '#/definitions/SsnPostBody'
+        - $ref: '#/parameters/osuId'
       responses:
-        202:
+        '202':
           description: SSN record accepted.
           schema:
-            $ref: '#/definitions/SSNResultObject'
-        400:
-          description: Bad request
-          schema:
-            $ref: '#/definitions/ErrorArray'
-        404:
-          description: Person not found
-          schema:
-            $ref: '#/definitions/Error'
-  /persons/{osuID}/image:
+            $ref: '#/definitions/SsnResult'
+        '400':
+          $ref: '#/responses/400'
+        '404':
+          $ref: '#/responses/404'
+        '409':
+          $ref: '#/responses/409'
+        '500':
+          $ref: '#/responses/500'
+  /persons/{osuId}/images:
     get:
-      description: |
-        Get an ID card image for a person. If the person exists but doesn't have
-        an ID card image, a default image will be returned.
+      description: Get a person's image
       tags:
         - images
-      operationId: getImage
+      operationId: getImageById
       produces:
         - image/jpeg
         - application/json
       parameters:
+        - $ref: '#/parameters/osuId'
         - in: query
           name: width
+          maximum: 2000
+          minimum: 1
+          default: 100
           description: Resizes image to a pixel value for width while keeping original aspect ratio
           required: false
           type: integer
-        - $ref: '#/parameters/osuID'
-        - $ref: '#/parameters/pretty'
-        - $ref: '#/parameters/authorization'
       responses:
-        200:
+        '200':
           description: Successful response
           schema:
             type: file
           headers:
-              Content-type:
-                type: string
-                description: image/jpeg
-        404:
-          description: Person not found
-          schema:
-            $ref: '#/definitions/Error'
-  /persons/{osuID}/meal-plans:
+            Content-type:
+              type: string
+              description: image/jpeg
+        '400':
+          $ref: '#/responses/400'
+        '404':
+          $ref: '#/responses/404'
+        '500':
+          $ref: '#/responses/500'
+  /persons/{osuId}/meal-plans:
     get:
       description: Get meal plans for a person
       tags:
         - meal-plans
       operationId: getMealPlans
       parameters:
-        - $ref: '#/parameters/osuID'
-        - $ref: '#/parameters/authorization'
+        - $ref: '#/parameters/osuId'
+        - name: filter[mealPlanName]
+          in: query
+          type: string
+          required: false
+          description: Filter results by meal plan name.
+        - name: filter[balance]
+          in: query
+          type: string
```

Comment:
I believe this is a number, and we will need operations for this parameter: https://github.com/osu-mist/style-guides/blob/master/OpenAPI.md#filter-query-parameters